pkg/apis/edgefs.rook.io/v1beta1/types.go

```diff
@@ -321,8 +321,8 @@ type ISGWSpec struct {
 	RemoteURL string `json:"remoteURL,omitempty"`
 	// ISGW Replication Type
 	ReplicationType string `json:"replicationType,omitempty"`
-	// ISGW Metadata Only flag
-	MetadataOnly bool `json:"metadataOnly,omitempty"`
+	// ISGW Metadata Only flag, all or versions
+	MetadataOnly string `json:"metadataOnly,omitempty"`
```

Comment:
Changing this from a bool to a string may cause issues during upgrade. If the serialization/deserialization is handled ok, do you still handle the previous values of true and false?

Reply:
This service didn't work with bool at all. That was essentially the root of the bug. It has to be string.